unlock-app/src/components/interface/locks/Settings/forms/UpdateGasRefundForm.tsx

```diff
@@ -45,7 +45,7 @@ export function UpdateGasRefundForm({ lockAddress, network, disabled }: Props) {
 
   useEffect(() => {
     if (!isLoading && gasRefundValue) {
-      setValue('amount', Number(gasRefundValue).toFixed(0))
+      setValue('amount', gasRefundValue)
```

Comment:
have you checked the associated unlock-js method to see if it works with decimal values properly? I remember ethers has some issues with parsing decimal values.

Reply:
I have tested a bunch and it does work as expected.
`ethers.utils.parseUnits(gasRefundValue, decimals)` is able to handle things with decimals.